### ISSUE TITLE
[Deeplab] Can we provide only COCO pretrained checkpoints? How well does Deeplab work for indoor scenes?

### ISSUE THREAD

**Issue by Harry-Zhi** [tensorflow/models#4039]
Hi,

In DeepLab model Zoo, models trained on PASCAL VOC 2012 and Cityscapes are provided. 

I am just wondering **if you could provide the COCO only pre-trained weights**, since I want to fine-tune on my own data, where COCO pre-trained model may be a better starting point rather than VOC and cityscape pre-trained ones because COCO has more general classes and objects.

By the way, **have you tested the Deeplab segmentation using indoor datasets such as NYUv2 or Sun3D datasets? Does Deeplab work still well on indoor scenes**?

Best,
Shuaifeng

**Comment by bhack** [comment]
For some indoor scenes check https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/ade20k.md

**Comment by Harry-Zhi** [comment]
Any Comments?

**Comment by YknZhu** [comment]
Hmm sorry we dont have plan to release coco-pretrained checkpoint at this time. As we will reinitialize all logits layer parameters while tuning on a different dataset, using a coco-pretrained checkpoints or pascal/cityscape checkpoints might not matter too much.

Deeplab surely work on indoor scenes (see eval metrics on pascal). Retraining deeplab on NYUv2 or Sun3D is surely welcome :)

**Comment by Harry-Zhi** [comment]
@bhack Thanks for your reply. ADE20k is a good option but it contains both outdoor and indoor scenes. I am a bit curious about the performance on the small scale indoor datasets such as NYU and SUN. But ADE20K is of course a good benchmark to test.

**Comment by Harry-Zhi** [comment]
@YknZhu Hi, thanks for your comments. Because intuitively I think the indoor samples from PASCAL are not quite the same as NYU (PASCAL has a black backgrounds while NYU ground truth assigns a label to each pixel). But anyway I will try to use the PASCAL checkpoints to fine-tune NYU. And any comments are also welcome.

**Comment by bhack** [comment]
If you will contribute a PR with NYU, SUNRGBD or Matterport scripts it is appreciated.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
